packages/amplication-data-service-generator/src/register-plugin.ts

```diff
@@ -41,9 +51,10 @@ async function* getPluginFuncGenerator(
  */
 const getAllPlugins = async (pluginList: Plugin[]): Promise<Events[]> => {
   if (!pluginList.length) return [];
+  const allPlugins = [...pluginList, ...defaultPlugins];
```

Comment:
it is better to check whether the default plugins are not listed already - to avoid duplicate execution of the same plugin